README.md

```diff
@@ -35,6 +35,7 @@ Custom React PropType validators that we use at Airbnb. Use of [airbnb-js-shims]
  - `sequenceOf`: takes 1 or more "specifiers": an object with a "validator" function (a propType validator), a "min" nonNegativeInteger, and a "max" nonNegativeInteger. If both "min" and "max" may be omitted, they default to 1; if only "max" is omitted, it defaults to Infinity; if only "min" is omitted, it defaults to 1.
  - `shape`: takes a shape, and allows it to be enforced on any non-null/undefined value.
  - `uniqueArray`: this prop must be an array, and all values must be unique (determined by `Object.is`). Like `PropTypes.array`, but with uniqueness.
+ - `uniqueArrayWithMapper`: `uniqueArray`, with an additional function parameter that allows for a non-standard unique calculation (`Object.is`). The function is applied to each element in the array, and the resulting values are compared using the standard unique calculation.
```

Comment:
Yep, that's what I meant. I'll edit this to be clearer. As for moving this functionality into `uniqueArrayOf`, that seems good as well. I can take some time to work on that in the next few days